gulpfile.js

```diff
@@ -160,21 +102,95 @@ gulp.task('generate-grids-topics', () => {
                     "igSelector": grid.igSelector
                 }
             }))
-            .pipe(gulp.dest(DOCFX_ARTICLES + grid.igPath));
+            .pipe(dest(DOCFX_ARTICLES + grid.igPath));
     }
-});
-
-gulp.task('build', [
-    'styles',
-    'cleanup',
-    'post-processor-configs',
-    'generate-grids-topics',
-    'build-site'
-]);
-
-gulp.task('build-travis', [
-    'styles',
-    'cleanup',
-    'post-processor-configs',
-    'generate-grids-topics'
-]);
\ No newline at end of file
+     setTimeout( () => eventEmitter.emit("finish"), 3000);
+     return eventEmitter;
+}
+
+const  styles = () => {
+  return src(`${DOCFX_TEMPLATE}/styles/sass/main.scss`)
+        .pipe(sass().on('error', sass.logError))
+        .pipe(
+            autoprefixer({
+                overrideBrowserslist: ['last 2 versions'],
+                cascase: false
+            })
+        )
+        .pipe(dest(`${DOCFX_TEMPLATE}/styles/css`));
+};
+
+const buildSite = async() => {
+     await shell.task(`docfx build ${DOCFX_CONF}`)(done =>  done());
```

Comment:
done callback is not necessary because there isn't any parameter for it.
```suggestion
     await shell.task(`docfx build ${DOCFX_CONF}`)();
```

Reply:
Done